web/lib/components/flashcards/PageAllSets.jsx

```diff
@@ -82,6 +82,9 @@ export function PageAllSets({ page }) {
           notify({ title: _e('flashcardset.createNewSetSuccessfully') })
           setIsDialogOpen(false)
         },
+        onError: (...args) => {
+          console.log(...args)
+        },
```

Comment:
`onError: console.log` prolly works (or not, recently React has do something with it).

Reply:
Seems like it won't work as suggested.
https://github.com/reactwg/react-18/discussions/96